flink-external-resources/flink-external-resource-gpu/src/test/java/org/apache/flink/externalresource/gpu/GPUDiscoveryScriptTest.java

```diff
@@ -19,41 +19,40 @@
 package org.apache.flink.externalresource.gpu;
 
 import org.apache.flink.util.OperatingSystem;
-import org.apache.flink.util.TestLogger;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import java.io.BufferedReader;
 import java.io.InputStreamReader;
 
-import static org.junit.Assume.assumeTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /** Tests for the gpu-discovery-common.sh. */
-public class GPUDiscoveryScriptTest extends TestLogger {
+class GPUDiscoveryScriptTest {
 
     private static final String TEST_SCRIPT_PATH = "src/test/resources/test-coordination-mode.sh";
 
     @Test
-    public void testNonCoordinationMode() throws Exception {
-        assumeTrue(OperatingSystem.isLinux());
+    void testNonCoordinationMode() throws Exception {
+        assertThat(OperatingSystem.isLinux()).isTrue();
```

Comment:
this should still use assume.

Reply:
@zentol  Thank you very much for the good catch. I'm so sorry for the rookie mistake. I'll check it carefully

Reply:
Done.